connector/src/main/scala/com/basho/riak/spark/writer/WriteConf.scala

```diff
@@ -31,4 +37,9 @@ object WriteConf {
       writeQuorum = conf.getInt(WriteQuorumProperty, DefaultWriteQuorum)
     )
   }
+  
+  def fromOptions(options: Map[String, String], conf: SparkConf): WriteConf = {
```

Comment:
Hm.. works perfectly for me:

object WriteConf {
  val WriteQuorumProperty = "spark.riak.output.wquorum"

  val DefaultWriteQuorum = 1

//  def fromSparkConf(conf: SparkConf): WriteConf = {
//    WriteConf(
//      writeQuorum = conf.getInt(WriteQuorumProperty, DefaultWriteQuorum)
//    )
//  }

  def apply(conf: SparkConf, options: Map[String, String] = Map.empty): WriteConf = {
    null
  }
}